### PR TITLE
Removed duplicate line in the Changelog (v3.0.0)

### PR DESCRIPTION
**Pull request by SolPier**
Well, pretty self explanatory

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -94,7 +93,7 @@
     * Prevent unhandled rejections from being thrown twice (once by the `RSVP` unhandled rejection system, and again by `Ember.onerror` if present).
 - [#15873](https://github.com/emberjs/ember.js/pull/15873) [BUGFIX] Update to `backburner.js@1.2.3` to prevent issues with swallowing errors thrown within `run.join` callbacks when `Ember.onerror` is present.
 - [#15848](https://github.com/emberjs/ember.js/pull/15848) [BUGFIX] Ensure helpers have a consistent API.
-- [#15849](https://github.com/emberjs/ember.js/pull/15849) [BUGFIX] Fix issue when observing a computed property that is clobbered during creation. 
+- [#15849](https://github.com/emberjs/ember.js/pull/15849) [BUGFIX] Fix issue when observing a computed property that is clobbered during creation.
```
Comment (Anadeidra) [inline CHANGELOG.md]: My bad, its my editor. I can rollback this aswell if needed.

**Comment by rwjblue** [comment]
Sorry for missing it, but can you comment inline for each that is being removed and mention where they are duplicated?

**Comment by Anadeidra** [comment]
Inline on github, not in the code I suppose.
No problem 😉
I checked for v3.0.0 already (I deleted the only duplicate I found), but i can go further.

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,15 +19,14 @@
...
 - [#16097](https://github.com/emberjs/ember.js/pull/16097) / [#16110](https://github.com/emberjs/ember.js/pull/16110) [CLEANUP] Remove `sync` runloop queue.
 - [#16099](https://github.com/emberjs/ember.js/pull/16099) [CLEANUP] Remove custom eventManager support.
 - [#16067](https://github.com/emberjs/ember.js/pull/16067) [BUGFIX] Fix issues with `run.debounce` with only method and wait.
 - [#16045](https://github.com/emberjs/ember.js/pull/16045) [BUGFIX] Fix double debug output
 - [#16050](https://github.com/emberjs/ember.js/pull/16050) [BUGFIX] Add inspect and constructor to list of descriptor exceptions
 - [#16080](https://github.com/emberjs/ember.js/pull/16080) [BUGFIX] Add missing modules docs for tryInvoke, compare, isEqual #16079
 - [#16084](https://github.com/emberjs/ember.js/pull/16084) [BUGFIX] Update `computed.sort` docs to avoid state leakage
-- [#16087](https://github.com/emberjs/ember.js/pull/16087) [BUGFIX] Ensure `App.visit` resolves when rendering completed. 
+- [#16087](https://github.com/emberjs/ember.js/pull/16087) [BUGFIX] Ensure `App.visit` resolves when rendering completed.
```
Comment (Anadeidra) [inline CHANGELOG.md]: Duplicated from line 22

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,15 +19,14 @@
 - [#16169](https://github.com/emberjs/ember.js/pull/16169) [BUGFIX] Fix various issues with descriptor trap.
 - [#16174](https://github.com/emberjs/ember.js/pull/16174) [BUGFIX] Enable _some_ recovery of errors thrown during render.
```
Comment (Anadeidra) [inline CHANGELOG.md]: May be duplicated: found 3 other times:
- 2.18.1 (line 66)
- 2.17.1, (line 85)
- 2.16.3, (line 115)

**Comment by Anadeidra** [comment]
Line 42 maybe duplicated aswell, found in:
- 2.18.0 (line 75)
- 2.16.3 (line 113)

File: CHANGELOG.md (hunk 3 above)
Comment (rwjblue) [reply CHANGELOG.md]: Hmm, that isn't duplication though. That is the same bugfix landing in each of those versions.

**Review by locks — APPROVED** [review]

**Pull request merged**